packages/notification-center-vue/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "allowUnreachableCode": false,
+    "allowSyntheticDefaultImports": true,
+    "declaration": true,
+    "declarationDir": "dist/types",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "noLib": false,
+    "lib": ["dom", "es2020"],
+    "module": "ESNext",
+    "target": "es2017",
```

Comment:
Again the targets are different. But I ignore if this is needed, on purpose, or my question about being aligned in all the monolith is out of place.